.gitignore

```diff
@@ -17,5 +17,8 @@ src/Dockerrun.aws.json
 .idea
 src/api/tests/TestCases.json
 src/api/store/ZIPCODE_FIXES.json
+**/**/migrations
```

Comment:
Reverting these three changes to .gitignore
The repository includes migration files and settings.py
Just please don't commit them if you don't mean to.